### ISSUE TITLE
dry-run now requires cores or jobs to be set

### ISSUE THREAD

**Issue by akhanf** [snakemake/snakemake#1065]
**Snakemake version**
6.5.1

**Describe the bug**
In previous versions (e.g. 6.4.0 and earlier), running a dry-run did not require the use of --jobs or --cores. This seems to have changed now, as running `snakemake -np` gives an error now.

This breaks many previous tutorials and likely many tests as well (it does for me at least), so I am hoping this change in behaviour was unintended and could be reverted?

**Logs**
```
(venv) [akhanf@gra799 gen_unfold_template]$ snakemake -np
Error: you need to specify the maximum number of CPU cores to be used at the same time. If you want to use N cores, say --cores N or -cN. For all cores on your system (be sure that this is appropriate) use --cores all. For no parallelization use --cores 1 or -c1.
```

**Minimal example**
`snakemake -np`

**Comment by johanneskoester** [comment]
Right. Not really a necessary change. I'll restore the old behavior.

**Comment by johanneskoester** [comment]
Fixed in commit 6539a2dd.

**Comment by corneliusroemer** [comment]
Was about to open that issue myself just now, happy for it to have been fixed :)
